scrapy/core/downloader/contextfactory.py

```diff
@@ -135,11 +135,11 @@ def load_context_factory_from_settings(settings, crawler):
             settings=settings,
             crawler=crawler,
         )
-        msg = """
-            '%s' does not accept `method` argument (type OpenSSL.SSL method,\
-            e.g. OpenSSL.SSL.SSLv23_METHOD) and/or `tls_verbose_logging` argument and/or `tls_ciphers` argument.\
-            Please upgrade your context factory class to handle them or ignore them.""" % (
-            settings['DOWNLOADER_CLIENTCONTEXTFACTORY'],)
+        msg = f"""
+        {str(settings['DOWNLOADER_CLIENTCONTEXTFACTORY'])} \
+        does not accept `method` argument (type OpenSSL.SSL method,\
+        e.g. OpenSSL.SSL.SSLv23_METHOD) and/or `tls_verbose_logging` argument and/or `tls_ciphers` argument.\
+        Please upgrade your context factory class to handle them or ignore them."""
```

Comment:
Can we take the opportunity to stop using triple quotes here? I expect them to mess up spaces when the message is displayed:

```suggestion
        msg = (
            f"{settings['DOWNLOADER_CLIENTCONTEXTFACTORY']} does not accept "
            "a `method` argument (type OpenSSL.SSL method, e.g. "
            "OpenSSL.SSL.SSLv23_METHOD) and/or a `tls_verbose_logging` "
            "argument and/or a `tls_ciphers` argument. Please, upgrade your "
            "context factory class to handle them or ignore them."
        )
```

(I also took the liberty of including some articles and a comma, and removing the unneeded `str()`)

Reply:
@Gallaecio,  fixed -> https://github.com/scrapy/scrapy/pull/5246/commits/9c6b2b916dfa7efa9bd736144c0a64d2b5beccbd